packages/paste-design-tokens/tokens/global/box-shadow.yml

```diff
@@ -23,15 +23,21 @@ props:
   shadow-focus-inverse:
     value: "{!offset-0} {!offset-0} {!offset-0} {!offset-20} {!palette-gray-0-transparent-40}"
     comment: Shadow for focus ring on interactive elements on inverse backgrounds.
-  shadow-border-stronger:
+  shadow-border-weaker:
     value: "{!offset-0} {!offset-0} {!offset-0} 1px {!palette-gray-20}"
     comment: Lighter shadow border for disabled inputs.
-  shadow-border-strong:
+  shadow-border-weak:
     value: "{!offset-0} {!offset-0} {!offset-0} 1px {!palette-gray-30}"
     comment: Light shadow border for disabled inputs.
   shadow-border:
     value: "{!offset-0} {!offset-0} {!offset-0} 1px {!palette-gray-50}"
     comment: Shadow border for inputs.
+  shadow-border-strong:
+    value: "{!offset-0} {!offset-0} {!offset-0} 1px {!palette-gray-80}"
+    comment: Dark shadow border for inputs.
+  shadow-border-stronger:
+    value: "{!offset-0} {!offset-0} {!offset-0} 1px {!palette-gray-90}"
+    comment: Darker shadow border for inputs.
```

Comment:
Let's remove shadow-border-stronger for now since there's no corresponding border token.